package.json

```diff
@@ -55,7 +55,6 @@
     "@types/lodash": "^4.14.194",
     "@types/luxon": "^3.1.0",
     "@types/node": "^16.11.41",
-    "@types/pino": "^7.0.5",
```

Comment:
We don't need any pino types anymore?

Reply:
No they're included in the pino package now.